src/components/TodoContainer.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import TodosList from './TodosList';
+import Header from './Header';
+import InputTodo from './InputTodo';
+import { v4 as uuidv4 } from 'uuid';
+class TodoContainer extends React.Component {
+  state = {
+    todos: [],
+  };
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/todos?_limit=10')
+      .then((response) => response.json())
+      .then((data) => this.setState({ todos: data }));
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.todos !== this.state.todos) {
+      const temp = JSON.stringify(this.state.todos);
+      localStorage.setItem('todos', temp);
+    }
+  }
+
+  componentDidMount() {
+    const temp = localStorage.getItem('todos');
+    const loadedTodos = JSON.parse(temp);
+    if (loadedTodos) {
+      this.setState({
+        todos: loadedTodos,
+      });
+    }
+  }
+
+  componentWillUnmount() {
+    console.log('Cleaning up...');
```

Comment:
Kindly remove all `console.log()`, it is JavaScript best practice to write code without `console.log`.